.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
@andrcuns good point. hm... I think both solutions have positive and negative side(s). e.g. having those submodules only for testing purpose do not make this repo looks clean in my opinion.